clippercard/parser.py

```diff
@@ -129,6 +129,11 @@ def parse_card_products(card_soup):
         for value_node in blings:
             name = list(value_node.previous_siblings)[1].get_text().strip(':')
             products.append(CardProduct(name=name, value=value_node.get_text()))
+        caltrain = card_section.find_all('div', text=re.compile('^Valid till'))
```

Comment:
double checking this is not a typo with an extra L: `till` vs. `til`

Reply:
The only Issue I can now see is that the new revision may not pickup pending products. 
The first iteration would pick up pending products but for products already added to the ClipperCard it would give them blank entry in the value column.